cider/exec/template/CodeGenerator.h

```diff
@@ -418,14 +424,24 @@ class CodeGenerator {
                                      const bool operand_is_const,
                                      const CompilationOptions& co);
 
+  // to be deprecated
   llvm::Value* codegenCastToFp(llvm::Value* operand_lv,
                                const SQLTypeInfo& operand_ti,
                                const SQLTypeInfo& ti);
 
+  llvm::Value* codegenCastToFpForArrow(llvm::Value* operand_lv,
```

Comment:
What's the main difference between original `codegenCastToFp` and new `codegenCastToFpForArrow`?

Reply:
Removed null value check and simplified type conversion code.